share/spice/dota2/content.handlebars

```diff
@@ -0,0 +1,149 @@
+{{#if isItem}}
+    <div style="float: left; width: 100px;">
+        <img src="{{image}}" title="{{name}}"/>
+    </div>
+    <div style="float: right; width: 500px;">
+        <h1 class="zci__header">
+            {{name}}
+        </h1>
+        <hr />
+        <div class="twenty text--secondary">
+            Cost
+        </div>
+        <div class="eighty">
+            <img src="http://cdn.dota2.com/apps/dota2/images/tooltips/gold.png" title="Gold"/> {{cost}}
+        </div>
+        {{#if description}}
+            <div class="twenty text--secondary">
+                Description
+            </div>
+            <div class="eighty">
+                {{{description}}}<br />
+            </div>
+        {{/if}}
+        {{#if notes}}
+            <div class="twenty text--secondary">
+                Notes
+            </div>
+            <div class="eighty">
+                {{{notes}}}<br />
+            </div>
+        {{/if}}
+        {{#if attributes}}
+            <div class="twenty text--secondary">
+                Attribues
+            </div>
+            <div class="eighty">
+                {{{attributes}}}<br />
+            </div>
+        {{/if}}
+        {{#if components}}
+            <hr />
+            <div class="twenty text--secondary">
+                Recipe
+            </div>
+            <div class="eighty">
+                <table>
+                    <tr>
+                        {{#each components}}
+                            <td style="padding-right: 5px; text-align: center; vertical-align: top; width: 90px;">
+                                <a href="/?q=dota2+{{searchTerms}}&ia=dota2">
+                                    <img src="{{img}}" title="{{name}}"/>
+                                </a>
+                            </td>
+                        {{/each}}
+                    </tr>
+                </table>
+            </div>
+        {{/if}}
+    </div>
+    <div style="clear: both;" />
+{{else}}
+    {{#if isHero}}
+        <div style="float: left; width: 210px;">
```

Comment:
If you prefer, you can actually create sub-templates in the share dir and use them here (or inside other templates) `include 'template_name`

> https://github.com/duckduckgo/duckduckgo-template-helpers/blob/ee1713f5218c18a880a133583415cb09d8088a1b/template_helpers.js#L363